waspc/src/Wasp/AI/GenerateNewProject/Plan.hs

```diff
@@ -195,55 +198,55 @@ generatePlan newProjectDetails planRules = do
                    ]
           return (False, fixedPlan)
 
-checkPlanForEntityIssues :: Plan -> [String]
-checkPlanForEntityIssues plan =
-  checkNumEntities
-    <> checkUserEntity
-    <> concatMap checkIfEntityPSLCompiles (entities plan)
+checkPlanForModelIssues :: Plan -> [String]
+checkPlanForModelIssues plan =
+  checkNumModels
+    <> checkUserModel
+    <> concatMap checkIfModelBodyParses (models plan)
   where
-    checkNumEntities =
-      let numEntities = length (entities plan)
+    checkNumModels =
+      let numModels = length (models plan)
           expectedNumEntities = 2
-       in if numEntities < expectedNumEntities
+       in if numModels < expectedNumEntities
```

Comment:
I just see now that you still have `expectedNumEntites` here. I did suggest however that we go back to `entities` in this file, so in that case no need to fix this one.